new-examples/graphql-service-basic-auth-file-user-store/graphql_service_basic_auth_file_user_store.description

```diff
@@ -0,0 +1,14 @@
+// An GraphQL service can be secured using Basic auth and optionally by
+// enforcing authorization. Then, it validates the Basic auth token sent in the
+// `Authorization` header against the provided configurations. This reads data
+// from a file, which has a TOML format. This stores the usernames, passwords
+// for authentication, and scopes for authorization.<br/>
+// Ballerina uses the concept of scopes for authorization. A resource declared
+// in a service can be bound to one/more scope(s).<br/>
+// In the authorization phase, the scopes of the service/resource are compared
+// against the scope included in the user store for at least one match between
+// the two sets.<br/>
+// `Config.toml` has defined three users - Alice, Bob and Eve. Each user has a
```

Comment:
These are different from the actual values, right? Shall we update them?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-distribution/pull/1816/commits/b30eee9eebf69efacbc40cf75017cf4bf36c2695